config/dotfiles/bashrc

```diff
@@ -33,3 +33,32 @@ elif [[ -f "$CONFIG_HOME" ]]; then
 else
   echo "$(tput setaf 1)Can't find a home or work config file$(tput sgr 0)"
 fi
+
+#######################################
+# Executes the specified command. Alerts the user when the command completes.
+# Arguments:
+#   Command to execute
+# Returns:
+#   None
+#######################################
+notify() {
+  local title="Succeeded"
+  # Capture the parameter list for later display.
+  local message=$@
+
+  if [[ $# -eq 0 ]]; then
```

Comment:
I moved from single `[` and `]` to double `[[` and `]]`. The former is used for `sh` compatibility, which is not important to me.